src/wp-includes/theme.php

```diff
@@ -737,7 +737,7 @@ function locale_stylesheet() {
  * @param string $stylesheet Stylesheet name.
  */
 function switch_theme( $stylesheet ) {
-	global $wp_theme_directories, $wp_customize, $sidebars_widgets;
+	global $wp_theme_directories, $wp_customize, $sidebars_widgets, $wp_registered_sidebars;
```

Comment:
Quick review: Document missing for the `$wp_registered_sidebars`

Reply:
Resolved in [ba0fa42](https://github.com/WordPress/wordpress-develop/pull/3893/commits/ba0fa4294637c45d98ae2e1f7607904250e1cdc2).